arbitrum-docs/how-arbitrum-works/bold/gentle-introduction.md

```diff
@@ -12,148 +12,266 @@ import PublicPreviewBannerPartial from './partials/_bold-public-preview-banner-p
-- Tools and frameworks for the smooth migration of existing validators and a seamless onboarding for new validators to use BOLD for their respective Arbitrum chains.
+- A comprehensive, third-party audit of the [BoLD source code](https://github.com/OffchainLabs/BoLD) to ensure the effectiveness and safety of the design.
+- Tools and frameworks for the smooth migration of existing validators and a seamless onboarding for new validators to use BoLD for their respective Arbitrum chains.
 - Monitoring stack for people to use to see ongoing challenges on the testnet
 - A mechanism for the community to pool funds together to bond capital to an assertion made by validators
-- The launch of a public bounty program for white hat auditors and security professionals to help test and secure the BOLD protocol design.
-- Proposing, to the Arbitrum DAO, that the BOLD protocol be adopted - first for Arbitrum Sepolia and then eventually for Arbitrum One and Arbitrum Nova.
-- Cutting a GA release of Nitro that enables BOLD validation.
+- The launch of a public bounty program for white hat auditors and security professionals to help test and secure the BoLD protocol design.
+- Proposing, to the Arbitrum DAO, that the BoLD protocol be adopted - first for Arbitrum Sepolia and then eventually for Arbitrum One and Arbitrum Nova.
+- Cutting a GA release of Nitro that enables BoLD validation.
+
+## Frequently asked questions about BoLD (FAQ):
+
+#### Q: How does bonding work?
+The entities responsible for posting assertions about Arbitrum state to Ethereum are called validators. If posting assertions were free, anyone could create conflicting assertions to always delay withdrawals by 14 days instead of 7. As such, Arbitrum requires validators to put in a “security deposit”, known as a bond, to be allowed to post assertions. Validators can withdraw their bond as soon as their latest posted assertion has been confirmed, and end their responsibilities. These bonds can be any ERC20 token and should be set to a large enough value (e.g. 200 WETH) to make it economically infeasible for an adversary to attack an Arbitrum chain and to mitigate against spam (that would otherwise delay confirmations). Requiring a high bond to post assertions about Arbitrum seems centralizing, as we are replacing a whitelist of validators with instead a system that requires a lot of money to participate in. To address this, there is a [contract](https://github.com/OffchainLabs/BoLD/blob/main/contracts/src/assertionStakingPool/AssertionStakingPoolCreator.sol) that anyone can use to deploy a bonding pool as a way of crowdsourcing funds from others who wish to help defend Arbitrum but who may not individually be able to put up the large upfront bond itself. The use of bonding pools, coupled with the fact that there can be any number of honest anonymous parties ready to defend Arbitrum, means that these high bond values do not harm decentralization.
```

Comment:
```suggestion
The entities responsible for posting assertions about Arbitrum state to Ethereum are called validators. If posting assertions were free, anyone could create conflicting assertions to always delay withdrawals by 14 days instead of 7. As such, Arbitrum requires validators to put in a “security deposit”, known as a bond, to be allowed to post assertions. Validators can withdraw their bond as soon as their latest posted assertion has been confirmed, and end their responsibilities. These bonds can be any ERC20 token and should be set to a large enough value (e.g. 200 WETH) to make it economically infeasible for an adversary to attack an Arbitrum chain and to mitigate against spam (that would otherwise delay confirmations). Requiring a high bond to post assertions about Arbitrum seems centralizing, as we are replacing a whitelist of validators with instead a system that requires a lot of money to participate in. To address this, there is a [contract](https://github.com/OffchainLabs/BoLD/blob/main/contracts/src/assertionStakingPool/AssertionStakingPoolCreator.sol) that anyone can use to deploy a bonding pool as a way of crowdsourcing funds from others who wish to help defend Arbitrum but who may not individually be able to put up the large upfront bond itself. The use of bonding pools, coupled with the fact that there can be any number of honest, anonymous parties ready to defend Arbitrum, means that these high bond values do not harm decentralization.
```